styles/style.css

```diff
@@ -573,12 +323,8 @@ h1, h2, h3, h4, h5, h6, p {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The property -moz-animation-fill-mode is compatible with -webkit-animation-fill-mode and should be included as well.](https://www.codacy.com/app/j-hasseriis/cepheus/file/1888804921/issues/source?bid=3085146&fileBranchId=3088807#l327)